tests/test_features.py

```diff
@@ -0,0 +1,88 @@
+        assert package_is_installed(prefix, "track_feature=1.0")
+        assert not package_is_installed(prefix, "feature")
+
+        # install package that depends on track_feature
+        conda_cli("install", f"--prefix={prefix}", "feature", "--yes")
+        assert package_is_installed(prefix, "track_feature=1.0")
+        assert package_is_installed(prefix, "feature=1.0")
+
+        # install a newer track_feature
+        conda_cli("install", f"--prefix={prefix}", "track_feature=2.0", "--yes")
+        assert package_is_installed(prefix, "track_feature=2.0")
+        assert package_is_installed(prefix, "feature=2.0")
+
+
+def test_install_track_features_downgrade(
+    solver_classic: None,  # features not supported in libmamba
+    test_recipes_channel: Path,
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+):
+    # install a newer track_feature
+    with tmp_env("track_feature=2.0") as prefix:
+        assert package_is_installed(prefix, "track_feature=2.0")
+        assert not package_is_installed(prefix, "feature")
+
+        # install package that depends on track_feature
+        conda_cli("install", f"--prefix={prefix}", "feature", "--yes")
+        assert package_is_installed(prefix, "track_feature=2.0")
+        assert package_is_installed(prefix, "feature=2.0")
+
+        # install an older track_feature
+        conda_cli("install", f"--prefix={prefix}", "track_feature=1.0", "--yes")
+        assert package_is_installed(prefix, "track_feature=1.0")
+        assert package_is_installed(prefix, "feature=1.0")
+
+
+def test_remove_features_upgrade(
+    solver_classic: None,  # features not supported in libmamba
+    test_recipes_channel: Path,
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+):
+    # install an older track_feature
+    with tmp_env("track_feature=1.0", "feature") as prefix:
+        assert package_is_installed(prefix, "track_feature=1.0")
+        assert package_is_installed(prefix, "feature=1.0")
+
+        # remove the track_feature1, expecting to see track_feature2 be installed instead
+        conda_cli(
+            "remove", f"--prefix={prefix}", "--features", "track_feature1", "--yes"
+        )
+        assert package_is_installed(prefix, "track_feature=2.0")
+        assert package_is_installed(prefix, "feature=2.0")
+
+
+def test_remove_features_downgrade(
+    solver_classic: None,  # features not supported in libmamba
+    test_recipes_channel: Path,
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+):
+    # install a newer track_feature
+    with tmp_env("track_feature=2.0", "feature") as prefix:
+        assert package_is_installed(prefix, "track_feature=2.0")
+        assert package_is_installed(prefix, "feature=2.0")
+
+        # remove the track_feature2, expecting to see track_feature1 be installed instead
+        conda_cli(
+            "remove", f"--prefix={prefix}", "--features", "track_feature2", "--yes"
+        )
```

Comment:
I think this is broken behavior. The solver refuses to downgrade.

The equivalent `install` command does work though:

```
conda_cli("install", f"--prefix={prefix}", "track_feature=1.0", "--yes")
```

So maybe we just need to make a note about the broken behavior and move on. It's not like the users are left without options. Hopefully not too many depend on this flag. Which begs the question: should we deprecate `--features` as a flag? It's only available on `conda remove` 🤷

Reply:
> @mbargull do you have any insights here?

Unfortunately not.
I've mostly worked with/around the `features` stuff in the way it's nowadays still used, i.e., with `track_features` used just to "weigh down" versions/builds.
I also know how `features` are (or at least were around 2019 +/- a year or so) handled in `conda.resolve` -- but I didn't much looked into how they were handled in explicit specs in `conda.core.solve` etc. ...

> I'd be happy to mark the failing test as xfail

If you wanted to check if that test case was handled as you expected in older versions, versions `4.3.34`, `4.5.12`, `4.7.12` would probably candidates to test.
In case it isn't already marked as deprecated, I'd certainly suggest to mark the `--features` CLI arg as pending removal.

Reply:
tested with `conda 4.5.12` and got the same `PackagesNotFoundError` error so perhaps this was never possible

I will mark this as xfail